docs/architecture/grpc-for-wcf-developers/protobuf-data-types.md

```diff
@@ -1,11 +1,184 @@
 ---
 title: Protobuf scalar data types - gRPC for WCF Developers
-description: TO BE WRITTEN
+description: The basic and well-known data types supported by Protobuf and gRPC in .NET Core
 author: markrendle
-ms.date: 09/02/2019
+ms.date: 09/09/2019
 ---
 
 # Protobuf scalar data types
 
+Protobuf supports a range of native scalar value types. This table lists them all with their equivalent C# type.
+
+| Protobuf Type | C# Type    | Notes |
+| ------------- | ---------- | ----- |
+| double        | double     |       |
+| float         | float      |       |
+| int32         | int        |       |
+| int64         | long       |       |
+| uint32        | uint       |       |
+| uint64        | ulong      |       |
+| sint32        | int        | 1     |
+| sint64        | long       | 1     |
+| fixed32       | uint       | 2     |
+| fixed64       | ulong      | 2     |
+| sfixed32      | int        | 2     |
+| sfixed64      | long       | 2     |
+| bool          | bool       |       |
+| string        | string     | 3     |
+| bytes         | ByteString | 4     |
```

Comment:
Is this last line reversed? I don't recognize ByteString as a C# type.

Reply:
It's a Protobuf type; it's explained in the note below it.